src/GRA.Controllers/ChallengesController.cs

```diff
@@ -18,6 +18,10 @@ namespace GRA.Controllers
 {
     public class ChallengesController : Base.UserController
     {
+        public const string StatusAll = "All";
```

Comment:
What are these strings? Do they need to be internationalized? Why are they in constants?

Reply:
Part of @iafb's changes

Reply:
Those constants are for the url routing parameter values, I don't think they need to be internationalized.